op-chain-ops/devkeys/devkeys.go

```diff
@@ -0,0 +1,116 @@
+package devkeys
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// Domain separates the usage of development keys by broad kind of usage.
+// E.g. a general user account, a L1 thing, a superchain thing, a L2 thing, etc.
+type Domain uint64
+
+const (
+	// DefaultKeyDomain is the domain for keys that get pre-funded in every chain.
+	// A 0 chain ID should be used for this domain.
+	DefaultKeyDomain Domain = 0
+	// L1OperatorKeyDomain is the domain for keys used in L1 ops
+	L1OperatorKeyDomain Domain = 1
```

Comment:
Can we abstract this different than L1 and L2? - specifically what happens when we have L3 for example

Reply:
Yes, simplified this, good call